inbm/dispatcher-agent/dispatcher/source/ubuntu_source_manager.py

```diff
@@ -105,7 +105,7 @@ def add(self, parameters: ApplicationAddSourceParameters) -> None:
         # Step 2: Add the source
         try:
             create_file_with_contents(
-                os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.file_name), parameters.sources
+                os.path.join(UBUNTU_APT_SOURCES_LIST_D, parameters.source_list_file_name), parameters.sources
             )
         except (IOError, OSError) as e:
             raise SourceError(f"Error adding application source list: {e}")
```

Comment:
When catching exceptions for file operations, log the exception message to aid in debugging. [medium]